### PR TITLE
[ZEPPELIN-6037] Unification of the ZeppelinConfiguration variable

### PR DESCRIPTION
**Pull request by ParkGyeongTae**
### What is this PR for?
I have noticed that the ZeppelinConfiguration variable varies across files. To maintain cleaner code, I would like to propose unifying the 'conf' and 'zConf' variables into a single variable.

### What type of PR is it?
Improvement

### Todos
* [x] - conf -> zConf
* [x] - zconf -> zConf
* [x] - zeppelinConfiguration -> zConf

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-6037

### How should this be tested?
* CI
* Build and run the Zeppelin project

### Questions:
* Does the license files need to update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by Reamer** [comment]
Thank you for your contribution. You can also adjust the test files.

**Comment by ParkGyeongTae** [comment]
@Reamer 
I added a test code to check if the `private final ZeppelinConfiguration` field in all Java code is named `zConf`. If there is any code that needs to be added or modified, please advise.

**Comment by Reamer** [comment]
I think the test class `ZeppelinConfigurationFieldTest.java` is unnecessary. Further fields for renaming can be found in various test classes.
Examples:
 - `zeppelin-zengine/src/test/java/org/apache/zeppelin/conf/ZeppelinConfigurationTest.java`
 - `zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/AbstractInterpreterTest.java`
 - `zeppelin-server/src/test/java/org/apache/zeppelin/rest/AbstractTestRestApi.java`

**Comment by Reamer** [comment]
Thank you very much for your contribution. I see you have found more locations. If you think you have found them all, please let me know so I can do the review.

**Comment by ParkGyeongTae** [comment]
@Reamer Please conduct the testing and review. :)

File: zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZeppelinClientIntegrationTest.java
Comment (Reamer) [inline zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZeppelinClientIntegrationTest.java:83]: Please correct the indentation.

```suggestion
    zConf = zepServer.getZeppelinConfiguration();
```

**Review by Reamer — COMMENTED** [review]
Looks great except for a small indentation.

**Review by Reamer — APPROVED** [review]
LGTM, we will wait for the CI again before the merge.

**Comment by JooHyukKim** [comment]
Wow this must've been a lot of work. Great job @ParkGyeongTae !

**Comment by ParkGyeongTae** [comment]
@JooHyukKim Your words are really encouraging. Thank you! :)

**Pull request merged**
